crutest/src/main.rs

```diff
@@ -3089,10 +3111,13 @@ async fn perf_workload(
             .collect();
 
     let mut read_buffers: Vec<Buffer> = (0..io_depth)
-        .map(|_| Buffer::new(blocks_per_io, ri.block_size as usize))
+        .map(|_| Buffer::new(blocks_per_io, ri.volume_info.block_size as usize))
         .collect();
 
-    let es = ri.extent_size.value;
+    let mut es = 0;
+    for sv in ri.volume_info.volumes.iter() {
+        es += sv.blocks_per_extent;
+    }
```

Comment:
This doesn't make sense to me; the sum of blocks-per-extent across all subvolumes isn't a meaningful value.

In a subvolume-flavored world, there is no single extent size (`es`), so we should just remove it.  Extent count `ec` remains meaningful, but should be a function on `VolumeInfo`, e.g.
```rust
impl VolumeInfo {
    /// Returns the total number of extent files in this volume
    fn extent_count(&self) -> usize {
        self.subvolumes.iter().map(|s| s.extent_count).sum().unwrap_or(0)
    }
}
```

Reply:
Yeah, for this I'm going to remove the whole crutest `perf` test completely, so this was just to make it compile.
It's all going away.